src/js/components/favorite/Favorite.jsx

```diff
@@ -58,7 +52,7 @@ export class Favorite extends Component {
 }
 
 Favorite.propTypes = {
-    darkTheme: PropTypes.bool,
     checked: PropTypes.bool,
+    className: PropTypes.oneOf(['', 'dark-yellow', 'dark-white']),
```

Comment:
I'm not really thrilled with it either. Are you proposing that the style should be applied via a class on a parent container?